src/components/TodayWeather/styles.js

```diff
@@ -0,0 +1,99 @@
+import styled, { css } from "styled-components";
+
+export const TodayWeatherWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  font-family: var(--font-family);
+`;
+
+export const TodayWeatherHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: var(--margin-size-small);
+`;
+export const TodayWeatherImage = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: var(--margin-size-small);
+  flex: 1.2;
+  padding-top: var(--padding-size-medium);
+
+  & img {
+    width: 20vh;
+  }
+
+  @media (max-width: 1023px) {
+    .today-weather__image img {
+      width: max(20vh, 10rem);
+    }
+  }
+`;
+
+export const SearchTrigger = styled.button`
+  font-family: var(--font-family);
```

Comment:
I'm seeing you using this here and there, shouldn't be better to just set in the GlobalStyles the fonts, so you don't need to use anywhere?